### PR TITLE
8290 ux review fixes

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#8290

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: legal-api/src/legal_api/utils/legislation_datetime.py [hunk 1]
```diff
@@ -54,6 +54,15 @@ def as_utc_timezone(date_time: datetime) -> datetime:
         """Return a datetime adjusted to the GMT timezone (aka UTC)."""
         return date_time.astimezone(pytz.timezone('GMT'))
 
+    @staticmethod
+    def format_as_report_string(date_time: datetime) -> str:
+        """Return a datetime string in this format (eg: `August 5, 2021 at 11:00 am Pacific time`)."""
+        hour = date_time.strftime('%I').lstrip('0')
+        # %p provides locale value: AM, PM (en_US); am, pm (de_DE); So forcing it to be lower in any case
+        am_pm = date_time.strftime('%p').lower()
```
Comment (andrepestana-aot) [inline legal-api/src/legal_api/utils/legislation_datetime.py:62]: %P does it lower case.

**Review by andrepestana-aot — CHANGES_REQUESTED** [review]

**Comment by andrepestana-aot** [comment]
Looks good. Just use `%P` instead of converting `%p` to lowercase, please.

File: legal-api/src/legal_api/utils/legislation_datetime.py (hunk 1 above)
Comment (vysakh-menon-aot) [reply legal-api/src/legal_api/utils/legislation_datetime.py:62]: Yes I agree but the documentation says it depends on locale value

**Review by argush3 — APPROVED** [review]

File: legal-api/src/legal_api/utils/legislation_datetime.py (hunk 1 above)
Comment (andrepestana-aot) [reply legal-api/src/legal_api/utils/legislation_datetime.py:62]: The documentation only mentions `%p`. `%P` is only for linux with glibc: https://linux.die.net/man/3/strftime

**Review by andrepestana-aot — APPROVED** [review]

File: legal-api/src/legal_api/utils/legislation_datetime.py (hunk 1 above)
Comment (andrepestana-aot) [reply legal-api/src/legal_api/utils/legislation_datetime.py:62]: ok. Don't worry about it.

**Pull request merged**